lib/cmd/fh3/appdata/export/start.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+var fs = require('fs')
+  , fhreq = require('../../../../utils/request.js')
+  , readline = require('readline')
+  , common = require('../../../../common');
+
+module.exports = {
+  'desc': 'Start a new data export job for the given application in the given environment',
+  'examples': [{
+    cmd: 'fhc addpata export start --appId=5ujx2eifvzaudq43nw4nmvcu --envId=live',
+    desc: 'Start a new data export job for the given application in the given environment'
+  }],
+  'demand': ['envId', 'appId'],
+  'alias': {},
+  'describe': {
+    'envId': 'Environment id',
+    'appId': 'Application id'
+  },
+  'preCmd': function (params, cb) {
+    var rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout
+    });
+
+    var question = [
+      'Do you want to stop the application first?',
+      'The application data might be outdated if you choose not to. (Y/n):'
+    ].join('');
+
+    rl.question(question, function (answer) {
+      params.stopApp = shouldStopApp(answer);
+      rl.close();
+      return cb(null, params);
+    });
+  },
+  'customCmd': function (params, cb) {
+    var url = 'api/v2/appdata/' + params.envId + '/' + params.appId + '/data/export';
+
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, {
+      stopApp: params.stopApp
+    }, 'Error exporting data: ', function (err, result) {
+      if (err) {
+        return cb(err);
+      }
+
+      return cb(null, 'Created a new export job with ID: ' + result);
```

Comment:
I think @pb82 Is adding the polling to a `status` command on another subtask, we could also change this command to transition into that if possible.